Tests/UnitsTests/MeasurementTests.swift

```diff
@@ -196,6 +196,64 @@ final class MeasurementTests: XCTestCase {
         )
     }
 
+    func testPercent() throws {
```

Comment:
Please add an additional test in the `DefinitionsTests.swift` and copy the formatting there.

Reply:
I don't see that file?

Reply:
Whoops - my bad. I meant `DefinitionTests.swift`:

https://github.com/NeedleInAJayStack/Units/blob/main/Tests/UnitsTests/DefinitionTests.swift